src/queries/voting-data.ts

```diff
@@ -20,45 +20,50 @@ const aragonEnv = {
  * @param id - id of the vote to query (I.E. 21)
  * @param votingContractAddress - voting contract to query -- defaults to the main sarcophagus DAO address
  */
-function formatVoteId(id: string | number, votingContractAddress: string = '0xf483c1f7858dd19915d0689d26cb3487fc90b640'): any {
-  const aragonVoteIdPrefix = `appAddress:${votingContractAddress}-vote:`
-  const aragonVoteId = aragonVoteIdPrefix + "0x" + Number(Number(id).toString(16)).toString(16)
-  return aragonVoteId
+function formatVoteId(
+  id: string | number,
+  votingContractAddress: string = '0xf483c1f7858dd19915d0689d26cb3487fc90b640'
+): any {
+  const aragonVoteIdPrefix = `appAddress:${votingContractAddress}-vote:`;
+  const aragonVoteId = aragonVoteIdPrefix + '0x' + Number(Number(id).toString(16)).toString(16);
+  return aragonVoteId;
 }
 
+/**
+ * Retrieves details of vote including address of voters and timing of execution
+ *
+ * @param web3 - blockchain provider
+ * @param voteId - id of the vote to query (I.E. 21)
+ */
 export async function fetchVoteData(web3: any, voteId: string | number): Promise<VotingData> {
-  const voteIdFormatted = formatVoteId(voteId)
+  const voteIdFormatted = formatVoteId(voteId);
 
   // Connect to aragon subgraph to retrieve voting data
-  const org = await connect(
-    aragonEnv.location,
-    'thegraph', 
-    { 
-      network: aragonEnv.network,
-      ethereum: web3.currentProvider
-    }
-  )
+  const org = await connect(aragonEnv.location, 'thegraph', {
+    network: aragonEnv.network,
+    ethereum: web3.currentProvider,
+  });
 
   // Grab the first 100 votes
   // TODO: will need to consider pagination as vote count exceeds 100
-  const voting = await connectVoting(org.app('voting'))
-  const votes = await voting.votes({ first: 100 })
+  const voting = await connectVoting(org.app('voting'));
+  const votes = await voting.votes({ first: 100 });
 
   // Filter the votes to only the provided vote ID
   // since aragon doesn't have a way to target a single vote with a query
   const vote = votes.find((v: any) => {
     return v.id === voteIdFormatted;
-  })
+  });
 
-  if(!vote) {
-    throw Error("Unable to retrieve vote: ${voteId}")
+  if (!vote) {
+    throw Error('Unable to retrieve vote: ${voteId}');
   }
 
-  const casts = await vote.casts()
+  const casts = await vote.casts();
```

Comment:
There is no need to use `as VotingData` here since it is implied by the return type of the function. 

The change on line 54 will cause this to throw errors since, `executedBlockNumber` expects a number and `vote.executedAt` is a string. Simply convert it to a number using `parseInt(vote.executedAt)`. Same for `snapshotBlockNumber`.